README.md

```diff
@@ -1,4 +1,22 @@
 # apps
-Coordinating writing apps on top of ipfs, and their concerns.
 
-This is a discussion repo. That means that all of the work gets done in the [issues](https://github.com/ipfs/apps/issues). Go check them out! 
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
+[![](https://img.shields.io/badge/project-IPFS-blue.svg?style=flat-square)](http://ipfs.io/)
+[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
```

Comment:
@RichardLitt works for me. Did you click the "join" link first?

![](https://cloud.githubusercontent.com/assets/24291/16168837/748b8c56-355a-11e6-8e36-59d25780202b.png)